docs/BUILD.md

```diff
@@ -16,30 +16,22 @@ The source (found in the `src` directory) is divided into several directories:
   * `platform` - Platform specific code for OS types, sockets and TLS.
   * `test` - Test code for the MsQuic API / protocol.
   * `tools` - Tools for exercising MsQuic.
-    * `attack` - Adversarial tool for exploiting protocol weaknesses.
-    * `etw` - Windows specific tool for processing MsQuic ETW logs.
-    * `interop` - Runs through the [IETF interop scenarios](https://github.com/quicwg/base-drafts/wiki/16th-Implementation-Draft).
-    * `ping` - Simple tool for gathering throughput measurements. Read more [here](../src/tools/ping/readme.md).
-    * `reach` - Tests for QUIC reachability of a server for several well-known ALPNs.
-    * `sample` - Minimal example of how to use the MsQuic API.
-    * `spin` - Randomly executes the MsQuic API to discover bugs.
-    * `void` - Randomly starts and stops the server to discover bugs.
 
-## PowerShell (6 or greater) Requirement
+# PowerShell Usage
 
-MsQuic uses several cross platform PowerShell build scripts to simplify build and test operations. PowerShell 6 or greater will need to be installed for them to work.
+MsQuic uses several cross-platform PowerShell scripts to simplify build and test operations. The latest PowerShell will need to be installed for them to work. These scripts are the **recommended** way to build and test MsQuic, but they are **not required**. If you prefer to use CMake directly, please scroll down to the end of this page and start with the **Building with CMake** instructions.
 
-### Install on Windows
+## Install on Windows
 
-You can install PowerShell 7.0 on Windows by running the following **PowerShell** script:
+You can install the latest PowerShell on Windows by running the following **PowerShell** script or read the complete instructions [here](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows).
 
 ```PowerShell
 iex "& { $(irm https://aka.ms/install-powershell.ps1) } -UseMSI"
 ```
 
-Then you will need to manually launch "PowerShell 7" to continue.
+Then you will need to **manually** launch "PowerShell 7" to continue. This install does not replace the built-in version of PowerShell.

```

Comment:
Delete `7` next to `PowerShell` (line 26 has changed to `the latest PowerShell`

Reply:
From my testing, the `7` is required:
![image](https://user-images.githubusercontent.com/20663557/81478007-91e23400-91cf-11ea-8dc1-7ae275097af2.png)